packages/fxa-admin-panel/src/components/PageRelyingParties/index.test.tsx

```diff
@@ -77,7 +77,7 @@ it('renders as expected with a relying party containing all fields', async () =>
   screen.getByText(MOCK_RP_ALL_FIELDS.redirectUri);
   screen.getByText(MOCK_RP_ALL_FIELDS.allowedScopes!);
   screen.getByText('1970', { exact: false });
-  expect(screen.getAllByText('Yes')).toHaveLength(3);
+  expect(screen.getAllByText('true')).toHaveLength(3);
```

Comment:
I felt like on this page given all the other data we display about RPs, `true` and `false` were nicer for these values. I can easily use the `BooleanResult` here to display "Yes" and "No" consistently if preferred, though.